CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,149 @@
+    <p align="center">
+    Last Revision - 22 Dec 2021
+    </p>
+</div>
+
+<br />
+
+### Purpose
+
+
+We want to make sustainable technology, this requires a community that is
+architected along the same principle. The Code of Conduct (CoC) is a key tool
+in ensuring a community that is diverse in experience, cohesive in practice,
+and growing in membership [^1] [^2]. We welcome feedback.
+
+<details><summary><b>Show more</b></summary><br />
+
+We empathize with serious concerns that CoC may represent a loss of freedom of
+speech, inhibit personal expression, or bring unnecessary discussion to what
+should be the development of technology [^3].
+
+To these concerns, we say that evidence-based research has shown that Codes
+of Conduct are instrumental to achieving and sustaining inclusive communities
+which promote diverse sets of skills and perspectives [^1] [^2] [^3] [^4]. Meritocracy is not
+sustainable outside of interactions between absolute equals, including cultural
+norms and understandings [^2].
+
+</details>
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Scope
+
+This Code of Conduct applies to spaces both online and offline.
+
+<details>
+
+Online spaces managed by the Nim-Works organization will be held to the
+following standards. Spaces that represent our users, or where users are
+otherwise representing Nim-Works, may still be held accountable to these
+standards depending upon context and official status of the representative.
+
+Offline spaces where members interact, including but not limited to
+conferences, will be similarly held to the standards below.
+
+</details>
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Code of Conduct
+
+Outlined below are the main values of the CoC, each of which can be expanded to
+show contributing rules and themes.
+
+1. <details><summary>We are an open and inclusive community, so I should behave in a manner which is respectful of others without discrimination.</summary><br />
+
+   1. Discrimination against an individual(s) or group(s) damages the community and attacks the goals of our contributions; we specifically will not discriminate based on:
+      1. Virtue of race or ethnicity; country of origin, or country of residence; gender(s), biological sex or sexual orientation; age or experience; economic standing or education; disability - physical or otherwise. Not listed are not excluded.
+   2. Harassment - online or otherwise are not tolerated. This includes any offensive, belittling or threatening behaviour toward an individual(s) or group(s).
+      1. Cultural diversity and language barriers may play a critical role in differentiating whether a perpetrator intentionally meant to be aggressive in their behaviour. Patience is crucial in investigating these cases.
+
+</details>
+
+2. <details><summary>Contribution does not entitle me or anyone else to act in any way that is not conducive to making this a productive and enjoyable environment</summary><br />
```

Comment:
```suggestion
2. <details><summary>Contribution does not entitle anyone to act in any way that is not conducive to making this a productive and enjoyable environment</summary><br />
```

Mostly a nit, I don't like how "me" is used here, it doesn't feel right, especially when we were using "we" earlier.

Reply:
While I can't remember the intention behind writing this as it is, I do agree with Saem's interpretation. I prefer it this way or rephrased as:

"Contribution does not entitle we, nor anyone else, to act in any way that is not conducive to making this a productive and enjoyable environment"